src/render.ts

```diff
@@ -164,408 +185,288 @@ export class ImageRenderer {
                 },
                 outline: {
                     name: 'off', params: {}
+                },
+                antialiasing: {
+                    name: 'off', params: {}
```

Comment:
I would add SMAA

Reply:
right, in that case you can try fxaa and see if it help a bit

Reply:
Switched to fxaa with [acf9b1c](https://github.com/molstar/molrender/pull/3/commits/acf9b1cf289c4df785f8ec57151a40e571cb6649).